tests/unit_tests/arelle/test_system_info.py

```diff
@@ -0,0 +1,18 @@
+"""Tests for system info."""
+from arelle.SystemInfo import get_system_info
+
+
+def test_get_system_info() -> None:
+    """Test that function get_system_info returns."""
+    function_result = get_system_info()
+
+    assert True
+    assert any(test_key in list(function_result.keys()) for test_key in [
+        "arch",
+        "docker",
+        "os_name",
+        "version_arelle",
+        "version_os",
+        "version_python",
+        "virtualenv",
+    ]) is True
```

Comment:
The `is True` is implicitly there so you can just remove it. You might restructure so that you can print out which one is there that isn't expected. I might restructure like:

```python
expected_keys = {
    "arch", 
    "docker", 
    "os_name", 
    "version_arelle", 
    "version_os", 
    "version_python", 
    "virtualenv"
}

resulting_keys = set(function_result.keys())

missing_keys = expected_keys - resulting_keys
assert len(missing_keys) == 0, f"Missing keys {missing_keys}"

extra_keys = resulting_keys - expected_keys
assert len(extra_keys) == 0, f"Extra keys {extra_keys}"
```

Reply:
Neat solution! Haven't used sets much in Python but this was a cool way of using them.

Reply:
I'll have to remove the last bit as an additional key is added on Mac environments. The resulting test should be good enough for this small function.